example/package.json

```
@@ -17,10 +17,10 @@
     "windows": "react-native run-windows --no-packager"
   },
   "peerDependencies": {
-    "react": "~16.11.0 || ~16.13.1 || ~17.0.1 || ~17.0.2",
-    "react-native": "^0.0.0-0 || 0.62 - 0.68 || 1000.0.0",
+    "react": "~16.11.0 || ~16.13.1 || ~17.0.1 || ~17.0.2 || ~18.0.0",
```

```suggestion
    "react": "~16.11.0 || ~16.13.0 || ~17.0.0 || ~18.0.0",
```
nit: since we use `~` we should just be able to use the .0 for the patch version, it will account for them all

---

In general, we shouldn't deviate from what react-native requires. By allowing the version number to go higher, we're basically allowing some leeway in case of emergencies. I don't think we should allow users to downgrade.

---

yeah that makes sense, in that case we can go 
```suggestion
    "react": "~16.11.0 || ~16.13.1 || ~17.0.0 || ~18.0.0",
```